Apps/WebClient/src/Server/Models/TourConfiguration.cs

```diff
@@ -0,0 +1,77 @@
+﻿// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.WebClient.Server.Models
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using HealthGateway.Database.Models;
+
+    /// <summary>
+    /// The Tour Configuration.
+    /// </summary>
+    public record TourConfiguration
+    {
+        /// <summary>
+        /// Gets or sets the tour update date time.
+        /// </summary>
+        public DateTime? LatestChangeDateTime { get; set; }
+    }
+
+    /// <summary>
+    /// Contains the ApplicationSettings values and mapping logic for the TourConfiguration.
+    /// </summary>
+    public static class TourSettingsMapper
```

Comment:
Feels like this is an overkill to me, can you consider adding the application setting key to be passed to the delegate (maybe optional)? It can also be made generic and parse/cast the type in the service or delegate call?

Reply:
I'm using the delegate as it was built. And add this for extensibility. I'll consider a simplification.